### DISCUSSION TITLE
Mono Repo Webhook Support

### DISCUSSION THREAD

**Discussion by jlengelbrecht** [coollabsio/coolify#3512] [Feature Requests]
Hi there. Love the app so far but my only gripe has to be the lack of mono repo support when it comes to webooks. I would like to keep all of my docker apps in a mono repo and have them update with renovate. 

Currently this is possible but any PR gets merged in triggers a redeploy of all applications. This for obvious reasons is not ideal. Would it be possible to add support for wehooks in the future that are aware of what changed during a PR and only update select application that actually changed rather then everything.  

Similar to how flux-cd operates when setup in a Kubernetes mono repo configurations. Only applications that have been included the PR would be redeployed.

**Comment by galer7** [comment]
I'm getting this `Watch Paths` field on version [v4.0.0-beta.341](https://github.com/coollabsio/coolify/releases/tag/v4.0.0-beta.341)

<img width="1050" alt="image" src="https://github.com/user-attachments/assets/5b48b79c-007c-40e6-8a56-413197a1286d">

Seems like it was added in this PR #1917

**Reply by jlengelbrecht** [reply]
Thank you for adding this. Would it be possible to also add this feature for the docker-compose section?
